sdk/monitor/monitor-opentelemetry-exporter/package.json

```diff
@@ -116,7 +116,6 @@
     "exporter",
     "cloud"
   ],
-  "type": "module",
```

Comment:
Could you share more why is this needed?

Reply:
Resolved the issue in the project by removing a `require` statement that was being ignored by the formatter. Should be good now.